azure/durable_functions/models/DurableOrchestrationClient.py

```diff
@@ -69,22 +70,25 @@ async def start_new(self,
         request_url = self._get_start_new_url(
             instance_id=instance_id, orchestration_function_name=orchestration_function_name)
 
-        response = await self._post_async_request(request_url, self._get_json_input(client_input))
+        response: List[SerializableToJSON] = await self._post_async_request(
```

Comment:
What was the conclusion about `SerializableToJSON`. I'm not sure about how Python feels about custom types like this to represent attributes of a type?

Reply:
I recently learned that the right approach here is to use typing Protocols [https://mypy.readthedocs.io/en/stable/protocols.html] which allows us to typecheck if some type exports some method. If so, we can implement `SerializableToJSON` as the union of some primitive types **plus** objects that export `to_json` + `from_json` methods.

However, that's a longer term goal. For now, I've changed `SerializableToJSON` back to `Any`, with the goal of merging this and revisiting the type in the future :)